lib/sisimai/bite/email/zoho.rb

```diff
@@ -0,0 +1,167 @@
+module Sisimai::Bite::Email
+  module Zoho
+    # Sisimai::Bite::Email::Zoho parses a bounce email which created by Zoho Mail.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Zoho.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'     => %r/mailer-daemon[@]mail[.]zoho[.]com\z/,
+        :'subject'  => %r{\A(?:
```

Comment:
__[rubocop]__
* Style/SymbolLiteral: Do not use strings for word-like symbol literals.

<!-- Comment by [SideCI](https://sideci.com) -->